src/amber/support/file_encryptor.cr

```diff
@@ -1,29 +1,37 @@
 require "./message_encryptor"
 
 module Amber::Support
-  module FileEncryptor
-    def self.encryption_key
-      ENV["AMBER_ENCRYPTION_KEY"]? || begin
-        if File.exists?(".encryption_key")
-          File.open(".encryption_key").gets_to_end.to_s
-        else
-          raise Exceptions::EncryptionKeyMissing.new
-        end
-      end
+  class FileEncryptor
```

Comment:
I really wish this hadn't changed. We've added complexity here without improving functionality.

Reply:
I make it a module again.